### ISSUE TITLE
Detroit: Become Human Demo (1224230)

### ISSUE THREAD

**Issue by Vladislavuz** [ValveSoftware/Proton#3917]
# Compatibility Report
- Name of the game with compatibility issues: Detroit: Become Human Demo
- Steam AppID of the game: 1224230

## System Information
- GPU: GTX 1060
- Driver/LLVM version: nvidia 415.27
- Kernel version: 4.19
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Vladislavuz/80ced5788bff17617ff96eb3b7a5536a
- Proton version: 5.0-7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1224230.log](https://github.com/ValveSoftware/Proton/files/4686905/steam-1224230.log)

## Symptoms 
Black screen after starting the game. 

## Reproduction
Launch the game.

**Comment by Newbytee** [comment]
For me, it gets to a screen asking whether I accept some terms, and then a seemingly never-ending black screen with all of my CPU cores sitting at max usage.

**Comment by artemyto** [comment]
Works on my end on Proton 5.0-7, but very slow (near 2 fps). But works great on Proton-GE 5.8: https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/5.8-GE-2-MF

Specs:

- Nvidia 1070, driver 440.66.08
- Ryzen 5 3600
- Archlinux, kernel 5.6.13

**Comment by Leopard1907** [comment]
Proton lacks upstream patches that makes Detroit to run performant.

https://bugs.winehq.org/show_bug.cgi?id=48389

https://source.winehq.org/git/wine.git/commitdiff/aceff2102296300010dc8db7db72f059a3204eeb

**Comment by DanMan** [comment]
https://bugs.winehq.org/show_bug.cgi?id=48291 sounds like it's very much a hack though and a generic solution isn't easy. As I understand, the game (just like RDR2) does some low level syscalls which are difficult to replicate/intercept or something. At least in a way that's still performant.

Discussion about kernel changes: https://lkml.org/lkml/2020/5/30/17

**Comment by Leopard1907** [comment]
@DanMan Yes , that is why a there is a proposal like this for Linux kernel now.

https://lore.kernel.org/lkml/50a9e680-6be1-ff50-5c82-1bf54c7484a9@gmail.com/T/#m9f74cfb8d6f5356a428e88b097ba3fda832179fa

**Comment by Vladislavuz** [comment]
> @DanMan Yes , that is why a there is a proposal like this for Linux kernel now.
> 
> https://lore.kernel.org/lkml/50a9e680-6be1-ff50-5c82-1bf54c7484a9@gmail.com/T/#m9f74cfb8d6f5356a428e88b097ba3fda832179fa

Making changes to the kernel is a long process, I guess. Could it be easier to make such changes to an existing SteamOS kernel?

**Comment by FabianK-Dev** [comment]
> Works on my end on Proton 5.0-7, but very slow (near 2 fps). But works great on Proton-GE 5.8: https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/5.8-GE-2-MF
> 
> Specs:
> 
>     * Nvidia 1070, driver 440.66.08
> 
>     * Ryzen 5 3600
> 
>     * Archlinux, kernel 5.6.13

Thanks for the workaround using Proton-GE 5.8! :heart: Works awesome! :smile:

**Comment by ByCybernetik** [comment]
The game sees the Dualshock 4 gamepad as an Xbox One Gamepad
